github_connector_odoo/models/odoo_module_version.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from docutils.core import publish_string
+
+from openerp import models, fields, api, _
+from openerp.tools import html_sanitize
+from openerp.addons.base.module.module import MyWriter
+from openerp.tools.safe_eval import safe_eval
+
+_logger = logging.getLogger(__name__)
+
+
+class OdooModuleVersion(models.Model):
+    _name = 'odoo.module.version'
+    _order = 'name, technical_name'
+
+    # Constant Section
+    _SETTING_OVERRIDES = {
+        'embed_stylesheet': False,
+        'doctitle_xform': False,
+        'output_encoding': 'unicode',
+        'xml_declaration': False,
+    }
+
+    _ODOO_TYPE_SELECTION = [
+        ('verticalization', 'Verticalization'),
```

Comment:
It doesn't. Am I correctly understanding that this is a high level classification of module types? We call it `Vertical Interest` on our website, but that's honestly always bugged me. What about `Vertical Integration`?

Also - I would love to see this as a model of its own so that the user can classify their own repos.